test/functional/test_framework/blocktools.py

```diff
@@ -223,6 +223,11 @@ def create_raw_transaction(node, txid, to_address, *, amount):
             signed_psbt = wrpc.walletprocesspsbt(psbt)
             psbt = signed_psbt['psbt']
     final_psbt = node.finalizepsbt(psbt)
+    if not final_psbt["complete"]:
+        node.log.info(f'final_psbt={final_psbt}')
+        for w in node.listwallets():
+            wrpc = node.get_wallet_rpc(w)
+            node.log.info(f'listunspent={wrpc.listunspent()}')
```

Comment:
22149: NOTE: this code was later removed in 22308

Reply:
but 22308 is not backported yet - so, that's fine, isn't it?

Reply:
yeah, I just made a note to myself while reviewing it cause it says "temporary" :)